analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ArrayPassedAsParams.cs

```diff
@@ -71,3 +71,25 @@ public class IndexerClass
 {
     public int this[params int[] i] => 1;
 }
+
+public class Repro6894
+{
+    //Reproducer for https://github.com/SonarSource/sonar-dotnet/issues/6894
+
+    public void Method(params object[] args) { }
```

Comment:
I think `params Array[]` causes also FPs. Can you add some test cases?
```suggestion
    public void Method(params object[] args) { }
    public void MethodArray(params Array[] args) { }
```